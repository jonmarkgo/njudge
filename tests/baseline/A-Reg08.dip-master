control   001       S1801MX   1 1 1 0 2 18 7 0 0 0 0
Process   Thu Jan  8 00:00:00 2004 (1073520000)
Start     Wed Dec 31 23:59:59 2003 (1072915199)
Ontime rat. min 0.000
T = resignation ratio max 1.000
Yet_Applied_deadline? 0
-
