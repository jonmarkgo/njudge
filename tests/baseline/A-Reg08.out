Register player 8
===== mail sent to player8@localhost/Re: XXXX 
:: Judge: XXXX  

Newuser package is not being sent.  To get it, send 'get package' to the Judge.
Registration processed.
===== end of mail ======
